server/graphql/schema/index.ts

```diff
@@ -1,16 +1,20 @@
 import { GraphQLObjectType, GraphQLSchema } from "graphql";
 
 // Queries
-import { getUsers } from "./queries/getUsers";
-import { getLeagues } from "./queries/getLeagues";
-import { getLeague } from "./queries/getLeague";
-import { getTeams } from "./queries/getTeams";
+import { getUsers } from "./queries//getUsers";
```

Comment:
I fixed this but you merged it wrong; 
double slash;
./queries**//**getUsers

Reply:
fixed